database/seeds/eggs/source-engine/egg-garrys-mod.json

```diff
@@ -49,6 +49,51 @@
             "user_viewable": 0,
             "user_editable": 0,
             "rules": "required|string|max:20"
+        },
+        {
```

Comment:
Won't this break if you don't provide a workshop ID?

Reply:
It'll spit an error but won't stop startup of the server